server/src/international_application_processor/onboarding_messages.go

```diff
@@ -0,0 +1,327 @@
+			interested in helping you organize a chapter. Once you have
+			identified them, reach out to me by email at %v to schedule a call.
+			Don’t hesitate to email me with any questions in the meantime.
+			I’m looking forward to hearing back from you,
+		</p>`, californiaCoordinator.Address)
+
+	body.WriteString(almiraTannerSignatureHtml)
+
+	msg.BodyHTML = body.String()
+
+	return msg, nil
+}
+
+func (b *onboardingEmailMessageBuilder) nonCaOrganizerNotNearAnyChapter() (mailer.Message, error) {
+	var msg mailer.Message
+	msg.FromName = globalCoordinator.Name
+	msg.FromAddress = globalCoordinator.Address
+	msg.ToName = b.fullName
+	msg.ToAddress = b.email
+	msg.Subject = "Getting involved with Direct Action Everywhere"
+
+	var body strings.Builder
+	fmt.Fprintf(&body, `<p>Hi %v,</p>`, html.EscapeString(b.firstName))
+
+	body.WriteString(`
+		<p>
+			Thank you for your interest in joining the DxE Network.
+			I am part of the international coordination (IC) team and I am here
+			to help you start a DxE chapter in your area. Our onboarding process
+			involves four key steps:
+		</p>
+		<ol>
+			<li>
+				Finding 1-2 other people in your area who are interested in
+				helping start the chapter, and then setting up a call with a
+				member of the IC team. During this call, we will explain the
+				whole onboarding process and you will also be assigned a mentor.
+			</li>
+			<li>
+				Completing 5 short training sessions that cover important
+				information about DxE and how to organize your first events.
+			</li>
+			<li>
+				Organizing your first action and community event. Don’t worry,
+				you will have a lot of time and support to make this happen!
+			</li>
+			<li>
+				Debriefing your action with your mentor and completing the
+				final onboarding steps for you and your chapter. At that point,
+				you will be an official DxE organizer in an official DxE
+				chapter!
+			</li>
+		</ol>
+
+		<p>
+			To begin, please find 1-2 other people in your area who are
+			interested in helping you start a DxE chapter. Once you have
+			identified them, reach out to me by email to schedule our first
+			call.
+			Don’t hesitate to email me with any questions.
+			I’m looking forward to hearing back from you.
+		</p>
+	`)
+
+	fmt.Fprintf(&body, `
+		<p>
+			%v<br/>
+			International Coordinator
+			Direct Action Everywhere
+		</p>`,
+		globalCoordinator.Name)
+
+	msg.BodyHTML = body.String()
+
+	return msg, nil
+}
+
+const networkMemberProgramIntroHtml = `
+		<p>
+			Thank you for your interest in joining the DxE Network.
+			We recently launched the Network Member Program which is a platform
+			for people who are interested in taking action with DxE but don’t
+			have a chapter near them or have the capacity to organize a chapter
+			themselves.
+			The only steps to complete to officially become a member are:
+		</p>
+		<ol>
+			<li>Watch a video</li>
+			<li>Complete a short quiz</li>
```

Comment:
❓ are these supposed to be links?

Reply:
yes, waiting for Michelle. will do something here before merging.

Reply:
done